lib/src/rpc/dial.dart

```diff
@@ -128,6 +175,7 @@ Future<ClientChannelBase> _dialDirectGrpc(String address, DialOptions options, S
 }
 
 Future<ClientChannelBase> _dialWebRtc(String address, DialOptions options, String Function() sessionCallback) async {
+  allowedAddress = address;
```

Comment:
Again from above, what is `allowedAddress` for?

Reply:
leftover code i meant to remove. sorry! thanks for catching that :)